straxen/plugins/nveto_recorder.py

```diff
@@ -16,8 +16,8 @@
                  help="Pretrigger time before coincidence window in ns."),
     strax.Option('resolving_time_recorder_nv', type=int, default=600,
                  help="Resolving time of the coincidence in ns."),
-    strax.Option('nbaseline_samples_lone_records_nv', type=int,
-                 default_by_run=[(0, 10), (12684, 26)], track=True,
+    strax.Option('nbaseline_samples_lone_records_nv', type=tuple,
+                 default=('baseline_samples_nv', 'ONLINE', True), track=True,
```

Comment:
Would this not also be ("CMT_model", ('baseline_samples_nv', "ONLINE"), True)

Reply:
see above comment

Reply:
I still disagree, if we want to make the options to call CMT consistent, we should do it from the start. Especially for such low level things like baseline options. Perhaps you and @tunnell can come up with some consistent naming scheme to interface with CMT because I don't have time I'm afraid.

Reply:
Hi @jorana 
I don't think this PR neither GitHub is the forum to discuss this and since you brought it let me clarify that we have this naming convention because you wanted to keep the `to_pe_per_run` available in the first place. If we agree to disable this options we can work towards a simple naming convention for CMT